app/services/pipeline_report_service.rb

```diff
@@ -258,6 +258,8 @@ def compute_aggregate_scores(species_counts, genus_counts)
     species_counts.each do |tax_id, species|
       genus = genus_counts[species[:genus_tax_id]]
       # Workaround placeholder for bad data (e.g. species counts present in TaxonSummary but genus counts aren't)
+      # TODO: investigate why a count type appearing in species is missing in its genus. It's possible the data is
+      # only missing in local/staging and is present on prod.
```

Comment:
Please remove the last sentence... It is also missing in prod.
Did you add a Jira ticket?